### PR TITLE
QOLOE-122 correct handlebars loading by having an init function to wrap it

### PR DESCRIPTION
**Pull request by duttonw**
Other Changes:

* Removed doc's cloning dist
* Moved copying of storybook-static to github actions to save developer ssd's
* Created a new ./components/handlebars.helpers.bundle.js in the hbs folder
* Setup ./components/handlebars.init.js for squiz server side loading in same folder as hbs folder 
*  - This is un-minified, so you must load Handlebars library externally prior to call this.
* handlebars v4.7.8 added as dev dependancy for wiring, we may have been dependancy loading off bootstrap prior.

### CODE REVIEW

File: .esbuild/plugins/qgds-plugin-copy-assets.js [hunk 1]
```diff
@@ -18,12 +18,14 @@ export default function copyPlugin() {
         to: ["./dist/sample-data/"],
       },
       {
-        from: ["./src/js/handlebars.helpers.js"],
+        from: ["./src/js/handlebars*"],
         to: ["./dist/assets/js"],
       },
+      {
+        from: ["./src/js/handlebars*"],
+        to: ["./dist/components/"],
```
Comment (jortronQGov) [inline .esbuild/plugins/qgds-plugin-copy-assets.js:26]: Are copies needed in both /assets/js and /components folders?

File: esbuild.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const argv = minimist(process.argv.slice(2));
 // https://esbuild.github.io/getting-started/#build-scripts
 const buildConfig = {
   bundle: true,
-  minify: true,
+  minify: false,
```
Comment (jortronQGov) [inline esbuild.js:23]: Does this mean all our outputs won't be minifed?

File: esbuild.js [hunk 3]
```diff
@@ -41,9 +41,13 @@ const buildConfig = {
       out: "./assets/css/qld.bootstrap",
     },
     {
-      in: "./src/js/handlebars.helpers.js",
+      in: "./src/js/handlebars.init.js",
       out: "./assets/js/handlebars.helpers.bundle",
     },
+    {
+      in: "./src/js/handlebars.init.js",
+      out: "./components/handlebars.helpers.bundle",
```
Comment (jortronQGov) [inline esbuild.js:49]: Again - does this need to be in both locations?

File: esbuild.js (hunk 2 above)
Comment (duttonw) [reply esbuild.js:23]: opps, will fix, this was for testing.

File: esbuild.js (hunk 3 above)
Comment (duttonw) [reply esbuild.js:49]: the assets is what goes to cdn. the components is server side rendering. or included in the spa app. 

@stvp-qld , @MitchAnderson93 , your thoughts?

Comment (stvp-qld) [reply esbuild.js:49]: /assets/js is good enough for Squiz use case.  Squiz will have a local copy of /assets/js on it's server due the gitbridge (cloned repo).  So it's version of server side rendering will reference handlebars jS at that path.

But for other use cases, ie if another  project wants to populate HBS variables server side, before sending to client, you'll need the helpers server side also.  Are they shipped as dependency with npm?

**Review by stvp-qld — APPROVED** [review]

**Pull request merged**